skipper.yaml

```diff
@@ -46,6 +46,7 @@ env:
   NAMESPACE: $NAMESPACE
   IMAGE_BUILDER: $IMAGE_BUILDER
   CONNECTIVITY_CHECK_IMAGE: $CONNECTIVITY_CHECK_IMAGE
+  API_VIP_CONNECTIVITY_CHECK_IMAGE: $API_VIP_CONNECTIVITY_CHECK_IMAGE
```

Comment:
it is enough to set it here, no need for change in update_assisted_service_cm.py

Reply:
ok, removed from cm.